### ISSUE TITLE
Error in section 7.4 (page 159)

### ISSUE THREAD

**Issue by edwardforgacs** [ragtux/cnility#5]
The second code example on page 159 incorrectly repeats the first.

It should read:

```
int day, month, year;

scanf("%d/%d/%d", &month, &day, &year);
```

**Comment by RisingThumb** [comment]
See #2 . This project is inactive

**Comment by edwardforgacs** [comment]
Yes. I was hoping the original author may see this and respond to the source request, as it is strange they seem to have disappeared (although have made a few other contributions in other repos in the last year).
